docs/app/data/components-page.json

```diff
@@ -709,10 +709,26 @@
             "title": "Notifications",
             "link": "notifications",
             "sections": [
+                {
```

Comment:
Add dependency in here, they will need to add the brower animation module to their app module - would also be good to mention this to the documentation